lib/vsc/mympirun/mpi/intelmpi.py

```diff
@@ -97,15 +98,19 @@ def make_mpdboot_options(self):
             mpd = which(['mpd.py'])
             self.mpdboot_options.append('-m "\\\"%s --bulletproof\\\""' % mpd)
 
+    def set_impi_tmpdir(self):
+        """Set location of temporary directory that Intel MPI should use."""
+        # this one also needs to be set at runtime
+        impi_tmpdir = tempfile.gettempdir()
```

Comment:
i'm going to merge this, but it is not a garantee that this is unique across all nodes

Reply:
I think Intel MPI takes care of that by itself, it just wants a place to use as a scratch location?